force-app/main/default/lwc/pearEventManager/pearEventManager.js

```diff
@@ -0,0 +1,69 @@
+import { LightningElement, api } from 'lwc';
+import { subscribe, unsubscribe, onError}  from 'lightning/empApi';
+import { ShowToastEvent } from 'lightning/platformShowToastEvent';
+
+export default class PearEventManager extends LightningElement {
+
+    subscription = {};
+     @api channel = '/event/AssetCancelInitiatedEvent';
```

Comment:
This only works for cancel, and not renew?

Reply:
Yes, This is just for the Cancel.

Reply:
Why don't you want to subscribe to renewal events?